diesel/src/pg/connection/mod.rs

```diff
@@ -15,7 +15,10 @@ use self::stmt::Statement;
 use crate::connection::*;
 use crate::deserialize::FromSqlRow;
 use crate::expression::QueryMetadata;
-use crate::pg::{metadata_lookup::PgMetadataCache, Pg, PgMetadataLookup, TransactionBuilder};
+use crate::pg::{
+    metadata_lookup::{GetPgMetadataCache, PgMetadataCache},
+    Pg, TransactionBuilder,
+};
```

Comment:
I would prefer to keep the imports in line with all other imports in diesel
```suggestion
use crate::pg::metadata_lookup::{GetPgMetadataCache, PgMetadataCache};
use crate::pg::{Pg, TransactionBuilder};
```